packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -1,39 +1,59 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tile-background-color: Specifies the background color of the component.
+ * @prop --calcite-tile-border-color: Specifies the border color of the component.
+ * @prop --calcite-tile-border-color-active: Specifies the border color of the component when in an active state.
+ * @prop --calcite-tile-border-color-hover: Specifies the border color of the component when in a hover state.
+ * @prop --calcite-tile-description-text-color: Specifies the description text color of the component.
+ * @prop --calcite-tile-description-text-color-hover: Specifies the description text color of the component when in an active or hover state.
+ * @prop --calcite-tile-heading-text-color: Specifies the heading text color of the component.
+ * @prop --calcite-tile-heading-text-color-hover: Specifies the heading text color of the component when in an active or hover state.
+ * @prop --calcite-tile-icon-color: Specifies the icon color of the component.
+ */
+
 :host {
-  @apply bg-foreground-1
-    text-color-3
-    box-border
-    inline-block
-    select-none
-    duration-150
-    ease-in-out;
-
-  .container {
-    @apply pointer-events-none
-      grid
-      grid-cols-1;
-    gap: var(--calcite-spacing-md);
-  }
+  --calcite-tile-background-color: var(--calcite-color-foreground-1);
+  --calcite-tile-border-color: var(--calcite-color-border-2);
+  --calcite-tile-border-color-active: var(--calcite-color-brand);
+  --calcite-tile-border-color-hover: var(--calcite-color-brand);
+  --calcite-tile-description-text-color: var(--calcite-color-text-3);
+  --calcite-tile-description-text-color-hover: var(--calcite-color-text-2);
+  --calcite-tile-heading-text-color: var(--calcite-color-text-2);
+  --calcite-tile-heading-text-color-hover: var(--calcite-color-text-1);
+  --calcite-tile-icon-color: var(--calcite-color-text-3);
```

Comment:
Remove state specific tokens.

Instead do 
```css
:host(:hover) {
  --calcite-tile-border-color: var(--calcite-color-brand);
  --calcite-tile-description-text-color: var(--calcite-color-text-2);
  --calcite-tile-heading-text-color: var(--calcite-color-text-1);
}
```

Reply:
This has been addressed in #8691.  Thanks for the feedback!